packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -75,9 +81,8 @@ class Paginator extends React.Component {
     };
   }
 
-  // TODO: Resolve lint issues - https://github.com/cerner/terra-core/issues/1689
-  /* eslint-disable jsx-a11y/no-static-element-interactions, jsx-a11y/anchor-is-valid, jsx-a11y/no-noninteractive-tabindex */
   buildPageButtons(totalPages, onClick) {
+    const { intl } = this.props;
```

Comment:
Could this be moved into the object destructuring on the line below?

Reply:
Great catch @Blackop778. Resolved here https://github.com/cerner/terra-core/pull/2485/commits/d961db300072dd0a36b415437ab98ac82efaf35f